### ISSUE TITLE
Albums: Add option to disable ZIP download

### ISSUE THREAD

**Issue by HenrysCat** [photoprism/photoprism#3635]
Currently when I create a shared folder users can download single images and all images as zip, could this be split in to individual options? i.e. disable zip download but enable single image download or vice versa.

**Comment by lastzero** [comment]
In principle yes. However, the question is how to configure it: Globally? Per user? Per album?

**Comment by HenrysCat** [comment]
Personally I would be happy with it as a global setting, others may feel different.

**Comment by lastzero** [comment]
Do you want to disable it for admins as well? I guess the main use case would be to disable it for guests / link visitors only?

**Comment by HenrysCat** [comment]
Yes guests / link visitors

**Comment by raptor170** [comment]
Also would like to see this if possible, when downloading a full album,or even when selecting multiple pictures to download at once, to have them download as individual files and not as a zip. My home users would like to select and download and have their photos on their device without the need to unzip first

**Comment by lastzero** [comment]
I believe this should be configurable with the settings we added?

- https://docs.photoprism.app/getting-started/config-files/settings/#albums
- https://docs.photoprism.app/getting-started/config-files/settings/#downloads

**Comment by graciousgrey** [comment]
Yes, setting Disabled to true in the album [download settings](https://docs.photoprism.app/getting-started/config-files/settings/#albums)  disables the option to download complete albums as zip.
